torch/_decomp/__init__.py

```diff
@@ -440,6 +440,8 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.unbind,
             aten.unfold_backward,
             aten.unfold_copy,
+            aten._unsafe_masked_index,
+            aten._unsafe_masked_index_put,
             aten._unsafe_index,
```

Comment:
I think these should not be here. Removing these from here, should allow us to remove them from `decomps_to_exclude` as well.

Reply:
If CI agrees that we can remove these, could you also remove `_unsafe_index`?